codespell_lib/data/dictionary.txt

```diff
@@ -45769,28 +45917,67 @@ vitories->victories
 vitrole->vitriol
 vitrual->virtual
 vitrually->virtually
-vitual->virtual
+vitual->virtual, visual,
+vitualenv->virtualenv
+vitualisation->virtualisation, visualisation,
+vitualise->virtualise, visualise,
+vitualised->virtualised, visualised,
+vitualises->virtualises, visualises,
+vitualising->virtualising, visualising,
+vitualization->virtualization, visualization,
+vitualize->virtualize, visualize,
+vitualized->virtualized, visualized,
+vitualizes->virtualizes, visualizes,
+vitualizing->virtualizing, visualizing,
+vitually->virtually, visually,
+vituals->visuals
+viusal->visual
+viusalisation->visualisation
+viusalise->visualise
+viusalised->visualised
+viusalises->visualises
+viusalising->visualising
+viusalization->visualization
+viusalize->visualize
+viusalized->visualized
+viusalizes->visualizes
+viusalizing->visualizing
 viusally->visually
+viusals->visuals
 viusualisation->visualisation
 viwe->view
 viwed->viewed
 viweed->viewed
 viwer->viewer
 viwers->viewers
 viwes->views
+vizual->visual
 vizualisation->visualisation
 vizualise->visualise
 vizualised->visualised
+vizualises->visualises
+vizualising->visualising
 vizualization->visualization, virtualization,
 vizualize->visualize
 vizualized->visualized
+vizualizes->visualizes
+vizualizing->visualizing
+vizually->visually
+vizuals->visuals
 vlarge->large
 vlaue->value
 vlaues->values
 vlone->clone
 vloned->cloned
 vlones->clones
+vluable->valuable
+vluables->valuables
+vluation->valuation
+vluations->valuations
+vlue->value
```

Comment:
Again because **b**, **g** and **v** are neighbours on many keyboards, perhaps:
```
vlue->value, blue, glue,
```
I thought about adding `clue` as well, but I don't know if there are such typos out there.

Reply:
Added `vlue->blue` and similar. I didn't add `vlue->glue` because a similar typo (`vlucose->glucose`) doesn't exist. `vlue->clue` doesn't really exist.